packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { cx } from '@leafygreen-ui/emotion';
+import InfoWithCircleIcon from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import Tooltip from '@leafygreen-ui/tooltip';
+
+import { iconBaseStyles, iconThemeStyles } from './InfoSprinkle.styles';
+import { Align, InfoSprinkleProps, Justify } from './InfoSprinkle.types';
+
+export const InfoSprinkle = React.forwardRef<
+  HTMLSpanElement,
+  InfoSprinkleProps
+>(
+  (
+    {
+      darkMode: darkModeProp,
+      children,
+      baseFontSize,
```

Comment:
did we want to add a max-width prop?

Reply:
I don't think we need a prop; what would it do? I can add a max-width of 400px in the component.

Reply:
if the consumer wants to change the max width they can use the `className` prop to change it with CSS